dashboard/Dockerfile

```diff
@@ -30,6 +30,21 @@ server {
     listen 80;
     root /usr/share/nginx/html;
     absolute_redirect off;
+
+    # Clickjacking protection, not supported via CSP meta.
+    add_header Content-Security-Policy "frame-ancestors 'none'" always;
+
+    # Full CSP via header, if not using the meta tag.
```

Comment:
What does "if not using the meta tag" mean?

Reply:
I agree, and I think that should go in the documentation in a build/serve the dashboard section, where we can place the replacement script docs, etc. I'll do that in another PR.